### PR TITLE
Develop > Blockchains > Deployment > Obtain Coretime

### PR DESCRIPTION
**Pull request by 0xLucca**
Add Obtain Coretime Page under Develop > Blockchains > Deployment section

### CODE REVIEW

File: develop/blockchains/deployment/obtain-coretime.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+title: Obtain Coretime
+description: TODO
```
Comment (eshaben) [inline develop/blockchains/deployment/obtain-coretime.md:3]: Please add a meta description

File: develop/blockchains/deployment/obtain-coretime.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+---
+
+# Obtain Coretime
+
+## Introduction
+
+Securing coretime is essential for operating a parachain on Polkadot. This guide explains the different methods of obtaining coretime and walks through the necessary steps to get your parachain running.
```
Comment (eshaben) [inline develop/blockchains/deployment/obtain-coretime.md:10]: Why is Securing coretime essential for operating a parachain on Polkadot?

File: develop/blockchains/deployment/obtain-coretime.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+## Initial Setup Steps
+
+1. Reserve a `ParaID`
+
+    First, you need to reserve a unique identifier for your parachain:
+
+    1. Connect to the relay chain
+    2. Submit the `registrar.reserve` extrinsic
+
+    Upon success, you'll receive a registered `ParaID`
```
Comment (eshaben) [inline develop/blockchains/deployment/obtain-coretime.md:38]:
```suggestion
1. Reserve a unique identifier, `ParaID`, for your parachain:

    1. Connect to the relay chain
    2. Submit the `registrar.reserve` extrinsic

    Upon success, you'll receive a registered `ParaID`
```

File: develop/blockchains/deployment/obtain-coretime.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+2. Register Parachain Configuration
+
+    Register your parachain's essential information:
+
+    Submit the `registrar.register` extrinsic with the following parameters:
+
+    - `id` - your reserved `ParaID`
+    - `genesisHead` - your exported genesis state
+    - `validationCode` - your exported Wasm validation code
+
```
Comment (eshaben) [inline develop/blockchains/deployment/obtain-coretime.md:49]:
```suggestion
2. Register your parachain's essential information by submitting the `registrar.register` extrinsic with the following parameters:

    - **`id`** - your reserved `ParaID`
    - **`genesisHead`** - your exported genesis state
    - **`validationCode`** - your exported Wasm validation code

```

File: develop/blockchains/deployment/obtain-coretime.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+3. Initialize Parachain Collator
+
+    Start your parachain collator and begin synchronization with the relay chain during the onboarding process.
```
Comment (eshaben) [inline develop/blockchains/deployment/obtain-coretime.md:52]:
```suggestion
3. Start your parachain collator and begin synchronization with the relay chain
```

File: develop/blockchains/deployment/obtain-coretime.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+## Obtaining Coretime
+
+### Bulk Coretime
+
+Bulk coretime provides several advantages:
+
+- Monthly allocation of resources
+- Guaranteed block production slots (every 12 seconds, or 6 seconds with Asynchronous Backing)
```
Comment (eshaben) [inline develop/blockchains/deployment/obtain-coretime.md:61]:
```suggestion
- Guaranteed block production slots (every 12 seconds, or 6 seconds with [Asynchronous Backing](https://wiki.polkadot.network/docs/learn-async-backing#asynchronous-backing){target=\_blank})
```

File: develop/blockchains/deployment/obtain-coretime.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+- Priority renewal rights
+- Protection against price fluctuations
+- Ability to split and resell unused coretime
+
+To purchase bulk coretime:
+
+1. Access the Coretime system parachain
+2. Interact with the broker pallet
```
Comment (eshaben) [inline develop/blockchains/deployment/obtain-coretime.md:69]:
```suggestion
2. Interact with the Broker pallet
```

File: develop/blockchains/deployment/obtain-coretime.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+### On-demand Coretime
+
+On-demand coretime allows for flexible, as-needed block production. To purchase:
+
+1. Ensure your collator node is fully synchronized with the relay chain
+2. From the account that registered the `ParaID`, submit the `onDemand.placeOrderAllowDeath` extrinsic with:
+
+    - `maxAmountFor` - sufficient funds for the transaction
+    - `paraId` - your registered `ParaID`
```
Comment (eshaben) [inline develop/blockchains/deployment/obtain-coretime.md:85]:
```suggestion
    - **`maxAmountFor`** - sufficient funds for the transaction
    - **`paraId`** - your registered `ParaID`
```

**Review by eshaben — COMMENTED** [review]
Formatting review ✅

File: develop/blockchains/deployment/obtain-coretime.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+
+## Prerequisites
+
+Before obtaining coretime, ensure you have:
+
+- Developed your parachain runtime using the Polkadot SDK
+- Set up and configured a parachain collator for your target relay chain
+- Successfully compiled your parachain collator node
+- Generated and exported your parachain's genesis state
+- Generated and exported your parachain's validation code (Wasm)
```
Comment (nhussein11) [inline develop/blockchains/deployment/obtain-coretime.md:29]: would it make sense to link to the pages/section of our docs here?

File: develop/blockchains/deployment/obtain-coretime.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+
+## Initial Setup Steps
+
+1. Reserve a unique identifier, `ParaID`, for your parachain:
+
+    1. Connect to the relay chain
+    2. Submit the `registrar.reserve` extrinsic
```
Comment (nhussein11) [inline develop/blockchains/deployment/obtain-coretime.md:36]: would it make sense to link to the rust docs for this extrinsic?

File: develop/blockchains/deployment/obtain-coretime.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+After successfully obtaining coretime, your parachain will automatically start producing blocks at regular intervals.
+
+For current marketplaces and pricing, consult the [Coretime Marketplaces](https://wiki.polkadot.network/docs/learn-guides-coretime-marketplaces){target=\_blank} page on the Polkadot Wiki.
+
+### On-demand Coretime
+
+On-demand coretime allows for flexible, as-needed block production. To purchase:
+
+1. Ensure your collator node is fully synchronized with the relay chain
+2. From the account that registered the `ParaID`, submit the `onDemand.placeOrderAllowDeath` extrinsic with:
```
Comment (nhussein11) [inline develop/blockchains/deployment/obtain-coretime.md:76]: same here, add link to rust docs

**Review by nhussein11 — APPROVED** [review]
besides those details, it LGTM! 🚀

**Review by eshaben — APPROVED** [review]
LGTM!

**Pull request merged**